src/main/java/com/example/test/controllers/TestController.java

```diff
@@ -1,15 +1,30 @@
 package com.example.test.controllers;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
 
 @RestController
-@RequestMapping("/test")
+@RequestMapping("/myapp")
 public class TestController {
 
-    @GetMapping("/m2")
-    public String m2(){
-        return "m2";
+    @Autowired
+    private RestTemplate restTemplate;
+
+    String url= "https://endpoint.free.beeceptor.com/my/api/path/favicon.ico";
+    @GetMapping
+    public Object getApi(){
+       String response= restTemplate.getForObject(url, String.class);
```

Comment:
can we create a method that takes as argument method type "GET or POST or PUT or PATCH or ..." and the request body and if there is a request params (like postman)
and perform the API call with the specified attributes